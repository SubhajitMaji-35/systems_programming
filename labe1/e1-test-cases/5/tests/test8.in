Amoeba
Bacteria
Fungi
Virus
Algae
Archae
Helminths
