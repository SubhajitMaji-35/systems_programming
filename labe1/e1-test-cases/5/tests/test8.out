moba
aceria
uni
irs
lge
rcae
elinth
